docs/9_0/components/autocomplete.md

```diff
@@ -101,6 +101,7 @@ AutoComplete provides live suggestions while an input is being typed.
 | autoSelection | true | Boolean | Defines if auto selection of items that are equal to the typed input is enabled. If true, an item that is equal to the typed input is selected.
 | escape | true | Boolean | Defines if autocomplete results are escaped or not.
 | queryMode | server | String | Specifies query mode, valid values are "server" (default), "client" and "hybrid". [more](https://github.com/primefaces/primefaces/issues/5298)
+| dropdownTabindex | null | String | Position of the dropdown button in the tabbing order.
```

Comment:
Can you add regular `tabindex` too it looks like its missing from this doc.

Reply:
Strange enough it was present in de documentation of version 7 and 8.